source/common.c

```diff
@@ -352,6 +353,210 @@ void aws_secure_zero(void *pBuf, size_t bufsize) {
 #endif     // #else not windows
 }
 
+#define AWS_DEFINE_ERROR_INFO_COMMON(C, ES) [(C)-0x0000] = AWS_DEFINE_ERROR_INFO(C, ES, "aws-c-common")
+/* clang-format off */
+static struct aws_error_info errors[] = {
```

Comment:
Should there be a way to internationalize this?

Reply:
Probably, but I believe that to be out of scope for this PR, and likely for GA.